lib/rails_tracepoint_stack/tracer.rb

```diff
@@ -3,12 +3,14 @@
 
 module RailsTracepointStack
   class Tracer
+    # TODO: Tracer.new shoud return the tracer. Is weird to call Tracer.new.tracer
     def tracer
       @trace ||= TracePoint.new(:call) do |tp|
         next if RailsTracepointStack::TraceFilter.ignore_trace?(trace: tp)
 
         params = fetch_params(tp)
-        
+
+        # TODO: Add the support to custom format
```

Comment:
maybe, **in the future**, move this to an "Output" class, passing just the tracepoint instance and letting the Output class handle will be a good improvement. Thanks!